YouVsVirus/Assets/Scripts/Components/nonSpawnableSpace.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// The nonSpawnableSpace class serves the purpose to
+// defines regions on the map that are non-spawnable.
+// Attach cirlceColliders to the game object that define
+// space where nothing should spawn.
+// If you then call the function coordinatesAreSpawnable2D with
+// a coordinate vector it will return false if the coordinates 
+// lie within the circles and true if they lie outside.
+namespace Components
+{
+public class nonSpawnableSpace : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
```

Comment:
Maybe remove Start and Update if you are not using it? Or is that unity bad practice?